src/core/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'dvoytenko'}, {name: 'ampproject/wg-performance'}],
```

Comment:
Please add me as an explicit code reveiwer.

Reply:
aren't you already a code reviewer for the whole repo?

Reply:
Done.

Reply:
@samouri Looks like he meant the `required` prop which ensures that, even with another OWNER's approval, nothing in here can change without his ✅

Reply:
It's intentionally undocumented because it slows PR review process so we don't want it to be overused, but can be useful for gatekeeping

Reply:
> It's intentionally undocumented

This seems like a bad idea....